test/beast/http/field_compiles.cpp

```diff
@@ -0,0 +1,4 @@
+//
+// check that http/field.hpp compiles unaided
+//
+#include <boost/beast/http/field.hpp>
```

Comment:
The file is missing the copyright notice and license. You can look at field.cpp and copy that